src/Tabs/TabsWrapper.php

```diff
@@ -189,16 +189,36 @@ function twAddTab(tabsid, label, content) {
   return panelId;
 }
 
+var execute = false;
+var temp;
 // Add a new tab using an iframe loading a specified URL.
 function twAddFrameTab(tabsid, label, url) {
   var panelId = nextPanelId(tabsid);
   top.restoreSession();
-  twAddTab(
-    tabsid,
-    label,
-    "<iframe name='" + panelId + "' frameborder='0' class='w-100' style='height:94.5%' src='" + url + "'>Oops</iframe>"
-  );
-  return panelId;
+  if (label === "Fee Sheet") {
+    if (!execute) {
+      twAddTab(
+        tabsid,
+        label,
+        "<iframe name='" + panelId + "' frameborder='0' class='w-100' style='height:94.5%' src='" + url + "'>Oops</iframe>"
+      );
+      execute = true;
+      temp = panelId;
+      return panelId;
+    } else {
+      alert("Fee sheet tab is already opened");
```

Comment:
I suggest in here we replace legacy alert to the asyncAlertMsg,
It may look like a better than legacy alert.

Reply:
@stu01509 Should I use `setTimeOut()` ?